help/skills.txt

```diff
@@ -0,0 +1,21 @@
+ <- @@index|<<Back to Index>>@@
+
+
+##3                             === <<SKILLS>> ===
+
+<< You can improve your skills by increasing job level. Each job level
+ gives you 1 point to spend on basic skills.>>
+
+   ##4 <<Basic skills:>>
```

Comment:
we do not but cassy made them "alwaysVisible" in skills.xml so I kept them here as well